Project-Aurora/Project-Aurora/Profiles/Stardew Valley/GSI/Nodes/JournalNode.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Aurora.Profiles.StardewValley.GSI.Nodes {
+
+    public class JournalNode : AutoJsonNode<JournalNode>
+    {
+        public bool QuestAvailable;
+        public bool NewQuestAvailable;
```

Comment:
What's the difference between these two? QuestAvailable and NewQuestAvailable.

Reply:
QuestAvailable == you have a quest
NewQuestAvailable == you have a unviewed quest (is labeled as "NEW" in your Journal)